### PR TITLE
Windows installer (Change descriptions add helping)

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Fixes https://github.com/netdata/netdata/issues/18704

screen3
screen2

##### Test Plan

1. Compile the branch
2. Generate the installer.
3. Have a look in texts on screen and finish the installation.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by stelfrag** [comment]
@thiago The behavior has changed. The token, rooms etc used to be asked before the installation starts -- now it is after the installation

**Comment by stelfrag** [comment]
The help button to clarify proxy and insecure may raise further questions as now, the user, needs to check libcurl documentation (instead of learn)

File: packaging/windows/installer.nsi [hunk 1]
```diff
@@ -145,8 +146,14 @@ Function un.onInit
 !insertmacro SingleInstanceFile
 FunctionEnd
 
+Function ShowHelp
+Pop $0
+        MessageBox MB_ICONQUESTION|MB_OK "$\"Proxy URL$\" accepts any proxy configuration supported by libcurl.$\n$\n$\"Insecure connection$\" instructs libcurl to disable host verification.$\n$\n$\"Open Terminal$\" launches a terminal to run additional commands after the installation completes." IDOK endHelp
```
Comment (ilyam8) [inline packaging/windows/installer.nsi:151]:
```suggestion
        MessageBox MB_ICONQUESTION|MB_OK "$\"Proxy URL$\" set the proxy server address to use if your network requires one.$\n$\n$\"Insecure connection$\" disable verification of the server's certificate chain and host name.$\n$\n$\"Open Terminal$\" open MSYS2 terminal to run additional commands after installation." IDOK endHelp
```

File: packaging/windows/installer.nsi [hunk 2]
```diff
@@ -145,8 +146,14 @@ Function un.onInit
...
+        endHelp:
+FunctionEnd
+
 Function NetdataConfigPage
-        !insertmacro MUI_HEADER_TEXT "Netdata configuration" "Claim your agent on Netdata Cloud"
+        !insertmacro MUI_HEADER_TEXT "Netdata configuration" "Connect your Agent on your Netdata Cloud Space"
```
Comment (ilyam8) [inline packaging/windows/installer.nsi:156]:
```suggestion
        !insertmacro MUI_HEADER_TEXT "Netdata configuration" "Connect your Agent to your Netdata Cloud Space"
```

File: packaging/windows/installer.nsi [hunk 3]
```diff
@@ -156,29 +163,34 @@ Function NetdataConfigPage
 
         IfFileExists "$INSTDIR\etc\netdata\claim.conf" NotNeeded
 
-        ${NSD_CreateLabel} 0 0 100% 12u "Enter your Token and Cloud Room(s)."
-        ${NSD_CreateLabel} 0 15% 100% 12u "Optionally, you can open a terminal to execute additional commands."
+        ${NSD_CreateLabel} 0 0 100% 12u "Please enter your Space's Claim Token and the Room IDs you want the Agent "
+        ${NSD_CreateLabel} 0 12% 100% 12u "to be added to. If left empty, it will be added to the $\"All nodes$\" Room"
```
Comment (ilyam8) [inline packaging/windows/installer.nsi:167]:
```suggestion
        ${NSD_CreateLabel} 0 0 100% 12u "Enter your Space's Claim Token and the Room IDs you want "
        ${NSD_CreateLabel} 0 12% 100% 12u "to add the Agent. If no Room IDs are specified, the Agent will be added to the $\"All nodes$\" Room."
```

**Comment by thiagoftsm** [comment]
> @thiagoftsm The behavior has changed. The token, rooms etc used to be asked before the installation starts -- now it is after the installation

Hello @stelfrag ,

You were mixing two PRs. The PR that put at the end still was not [merged](https://github.com/netdata/netdata/pull/18649). :)

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**